contribs/gmf/src/directives/lidarpanel.js

```diff
@@ -0,0 +1,349 @@
+goog.provide('gmf.lidarPanelComponent');
+
+goog.require('gmf');
+goog.require('gmf.lidarProfile');
+/**
+* @suppress {extraRequire}
```

Comment:
Cane you write it on 1 line (not 3) please

Reply:
Fixed